docs/data-sources/scalr_workspace.md

```diff
@@ -36,6 +35,8 @@ All arguments plus:
 * `operations` - Boolean indicates if the workspace is being used for remote execution.
 * `terraform_version` - The version of Terraform used for this workspace.
 * `working_directory` - A relative path that Terraform will execute within.
+* `module_version_id` - ID of module version, in the format `modver-<RANDOM STRING>`.
+* `vcs_provider_id` - ID of vcs provider, in the format `vcs-<RANDOM STRING>`.
```

Comment:
@penja I guess these lines are supposed to be in `Argument Reference` section

Reply:
@brezitskiy No is not argument currently is impossible to filter ws by vcs_provider_id or module_version